README.md

```diff
@@ -251,6 +257,128 @@ Here's an example of the typical output to the console:
 15
 ```
 
+#### Generate a waveform
```

Comment:
It's unclear what the "Generate a waveform" example does.

Reply:
Fixed in f716731